docs/move.md

```diff
@@ -130,11 +133,35 @@ This can now be achieved with the following procedure:
 
 9. Delete the bootstrap cluster
 
-**Important Note** The move process should be done when the BMHs are
-in a steady state. BMHs should not be moved while any operation is on-going i.e.
-BMH is in provisioning state. This will result in failure since the IP of the
-BMH might change after the move and the DHCP-leases from the management cluster
-are not moved to target cluster.
+## Impotant Notes
+
+The following requirements are essential for the move process to run
+successfully:
+
+1.
+The move process should be done when the BMHs are in a steady state. BMHs
+should not be moved while any operation is on-going i.e. BMH is in provisioning
+state. This will result in failure since the IP of the BMH might change after
+the move and the DHCP-leases from the management cluster are not moved to target
+cluster.
```

Comment:
the bigger risk is that the interaction between IPA and ironic get broken since ironic's database might not be repopulated. What you said is valid and should be there, but we should also add that doing this would compromise the state of your cluster.

Reply:
Done. Thanks